spec/std/isa/isa/globals.isa

```diff
@@ -1161,15 +1161,15 @@ function effective_ldst_mode {
     # when the mode is M, loads and stores can be executed as if they were done from any other mode
     # with the use of mstatus.MPRV
     if (mode() == PrivilegeMode::M) {
-      if (CSR[misa].U == 1 && CSR[mstatus].MPRV == 1) {
+      if ((!MISA_CSR_IMPLEMENTED) || (CSR[misa].U == 1 )&& CSR[mstatus].MPRV == 1) {
```

Comment:
this needs a longer test, and be careful with how the logic is grouped. Using `A || B && C` could be confusing (and seems to be incorrect here).